install_config/install/advanced_install.adoc

```diff
@@ -2521,6 +2521,15 @@ If for any reason the installation fails, before re-running the installer, see
 xref:installer-known-issues[Known Issues] to check for any specific
 instructions or workarounds.
 
+[CAUTION]
+====
+The installer caches playbook configuration values for 10 minutes, by default. If you change any system, network, or inventory configuration, 
```

Comment:
If you can't change the setting, I'd remove ", by default"

s/configuration, then/configuration and then